src/main.js

```diff
@@ -35,112 +35,118 @@ startButton.addEventListener('click', startCountDown);
 logButton.addEventListener('click', logActivity);
 createNewActivityBtn.addEventListener('click', createNewActivity);
 
-function changeColor(button1, category1, button2, category2, button3, category3) {
-    if (button2 == undefined) {
-        button1.classList.add(category1);
-    } else if (button3 == undefined) {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-    } else {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-        button3.classList.remove(category3);
-    }
+function addClassProperty(button1, category1, button2, category2) {
```

Comment:
Not a huge fan of this function signature. I would encourage that you make this function single responsibility - adds a class to a single HTML element. Then you can just call it for each element you need to add a class to and not worry about tracking so closely which arguments are passed to the function.